chia/types/eligible_coin_spends.py

```diff
@@ -97,12 +211,138 @@ def get_deduplication_info(
                     )
                     # Update this mempool item's coin spends map
                     bundle_coin_spends[coin_id] = BundleCoinSpend(
-                        spend_data.coin_spend, spend_data.eligible_for_dedup, spend_data.additions, spend_cost
+                        coin_spend=spend_data.coin_spend,
+                        eligible_for_dedup=spend_data.eligible_for_dedup,
+                        eligible_for_fast_forward=spend_data.eligible_for_fast_forward,
+                        additions=spend_data.additions,
+                        cost=spend_cost,
                     )
                 duplicate_cost = spend_cost
                 # If we end up including this item, update this entry's cost
-                new_eligible_spends[coin_id] = DedupCoinSpend(current_solution, duplicate_cost)
+                new_dedup_spends[coin_id] = DedupCoinSpend(current_solution, duplicate_cost)
             cost_saving += duplicate_cost
         # Update the eligible coin spends data
-        self.eligible_spends.update(new_eligible_spends)
+        self.deduplication_spends.update(new_dedup_spends)
         return unique_coin_spends, uint64(cost_saving), unique_additions
+
+    async def process_fast_forward_spends(
+        self,
```

Comment:
the only use of `self` here is to get to `self.fast_forward_spends`. I believe it would be easier to unit test this function if you would make this a free function, and passing in the `fast_forward_spends` as a parameter instead

Reply:
`fast_forward_spends` only makes sense as an internal state to `EligibleCoinSpends` and current tests already show how easy it is to unit test that. This is consistent with `deduplication_spends`.